### DISCUSSION TITLE
Is it possible to automate the generation of an optimized panel using multiple gerber/kicad files?

### DISCUSSION THREAD

**Discussion by danyhm** [INTI-CMNB/KiBot#694] [General]
Hello,

I know it's possible to automate the panel generation for a single design. But is it possible to do this automatically for multiple files? Lets say we have 4 designs and need a different quantities for each design.

For example 

![IMG_20241015_204102_980.jpg](https://github.com/user-attachments/assets/0430c38e-9632-4529-b755-3b1393ae3266)

**Comment by set-soft** [comment]
Hi @danyhm !

Nope, you can't do it automatically. You need to write some code: https://github.com/yaqwsx/KiKit/issues/83

**Comment by danyhm** [comment]
@set-soft 
Thanks for the clarification!
I'll take a look. it doesn't seem very hard to do.
